lua/lazy/manage/task/git.lua

```diff
@@ -8,7 +8,7 @@ local M = {}
 M.log = {
   ---@param opts {updated?:boolean, check?: boolean}
   skip = function(plugin, opts)
-    if opts.updated and not (plugin._.updated and plugin._.updated.from ~= plugin._.updated.to) then
+    if plugin.pin or opts.updated and not (plugin._.updated and plugin._.updated.from ~= plugin._.updated.to) then
```

Comment:
This should be a separate condition like
```lua
if opts.check and plugin.pin then
  return true
end
```

This way regular log would still work. Only check has to be disabled.

Reply:
Done.